### PR TITLE
Review section about Barcode control.

### PR DESCRIPTION
**Pull request by mariaterzieva12**

### CODE REVIEW

File: controls/barcode/2d-barcodes/qrcode-overview.md [hunk 1]
```diff
@@ -17,31 +17,31 @@ A QR code uses four standardized encoding modes to efficiently store data&mdash;
...
+* Blank Space&mdash;Additionally, around each QR code, has an obligatory 4-modules-wide space area:
 
   ![qrcode blank space](images/barcode-2d-barcodes-qrcode-overview004.png)
 
-* **Data**&mdash;The data occupies all available modules that are not occupied by any of the formatting data segments. If the data is less than the capacity of the remaining modules, it is padded to ensure that all modules are used. Additionally, the data consists of the actual encoded data, entered by the user, and the error correction bits, calculated on that data.
+* Data&mdash;The data occupies all available modules that are not occupied by any of the formatting data segments. If the data is less than the capacity of the remaining modules, it's padded to ensure that all modules are used. Additionally, the data consists of the encoded data, entered by the user, and the error correction bits, calculated on that data.
```
Comment (yordan-mitev) [inline controls/barcode/2d-barcodes/qrcode-overview.md:44]: Don't modify every occurrence of "It is" that the ReDox tool suggests. This suggestion is valid only for cases where "it is" is used as a placeholder and serves a purely grammatical function without adding meaning, for example:

- It is believed by some people that the Earth is flat -> Some people believe that the Earth is flat.
- It is a common mistake to swap "they're" with "their" -> Swapping "they're" with "their" is a common mistake.

The same rule is true for "there is" and "there are". See the [Style Guide](https://docs.telerik.com/style-guide/using-simple-non-ambiguous-language#using-simple-non-ambiguous-language) for more examples.
 
In the sentence that you modify, "it" is a pronoun that refers to "the data" and this loads the pronoun "it" with a specific meaning (the data). So in this case, "it is" is not serving a purely grammatical function, it has meaning.

```suggestion
* Data&mdash;The data occupies all available modules that are not occupied by any of the formatting data segments. If the data is less than the capacity of the remaining modules, it is padded to ensure that all modules are used. Additionally, the data consists of the encoded data, entered by the user, and the error correction bits, calculated on that data.
```

File: controls/barcode/overview.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+description: Try now the Telerik Barcode for .NET MAUI delivering a set of options for creating and showing barcodes.
 position: 0
 previous_url: /controls/barcode/barcode-overview
 slug: barcode-overview
 ---
 
 # .NET MAUI Barcode Overview
 
-**Telerik UI for .NET MAUI Barcode** enables you to create and show barcodes. You can generate and visualize barcodes in a machine-readable format through the Barcode control by providing numeric or character data.
+Telerik UI Barcode for .NET MAUI enables you to create and show barcodes. You can generate and visualize barcodes in a machine-readable format through the Barcode control by providing numeric or character data.
```
Comment (yordan-mitev) [inline controls/barcode/overview.md:12]:
```suggestion
The Telerik UI for .NET MAUI Barcode enables you to create and show barcodes. You can generate and visualize barcodes in a machine-readable format through the Barcode control by providing numeric or character data.
```

File: controls/barcode/sizing-modes.md [hunk 3]
```diff
@@ -12,8 +12,8 @@ slug: barcode-sizingmodes
...
 
 * Manual&mdash;You can define the size of the smallest line or dot of the barcode through the `Module` property and, as a result, the other lines and dots multiply that size. The `Module` property is measured in device pixels.
-* Snap&mdash;In the snap sizing mode the code is stretched to the available size, but each line or dot is drawn with an exact number of pixels. As a result, the lines and dots seem sharp.
-* Stretch&mdash;In the stretch sizing mode the code is stretched so as to fit exactly the available size. Each line or dot size is calculated so they snap to the device pixels. In order for the lines to stay sharp and stretch the barcode to the available size, some of them are slightly thicker than others.
+* Snap&mdash;In the snap sizing mode the code is stretched to the available size, but each line or dot is drawn with an exact number of pixels. As a result, the lines and dots appear sharp.
+* Stretch&mdash;In the stretch sizing mode the code is stretch to fit the available size. Each line or dot size is calculated so they snap to the device pixels. for the lines to stay sharp and stretch the barcode to the available size, some of them are thicker than others.
```
Comment (yordan-mitev) [inline controls/barcode/sizing-modes.md:16]: Replacing passive voice with active voice requires you to rewrite the sentence, not just to replace "stretched" with "stretch".
```suggestion
* Stretch&mdash;In the stretch sizing mode, the code is stretched to fit the available size. Each line or dot size is calculated so they snap to the device pixels. for the lines to stay sharp and stretch the barcode to the available size, some of them are thicker than others.
```

**Review by yordan-mitev — COMMENTED** [review]
* Use the correct component name in the meta description of all articles: Telerik UI for .NET MAUI Barcode.
* Be careful with "it is". See the explanation in one of the comments below.

**Review by didiyordanova — APPROVED** [review]

**Review by didiyordanova — APPROVED** [review]

**Pull request merged**